frigate/object_processing.py

```diff
@@ -239,25 +244,17 @@ def get_jpg_bytes(
             best_frame = cv2.resize(
                 best_frame, dsize=(width, height), interpolation=cv2.INTER_AREA
             )
-
         if timestamp:
-            time_to_show = datetime.datetime.fromtimestamp(
-                self.thumbnail_data["frame_time"]
-            ).strftime("%m/%d/%Y %H:%M:%S")
-            size = cv2.getTextSize(
-                time_to_show, cv2.FONT_HERSHEY_SIMPLEX, fontScale=1, thickness=2
-            )
-            text_width = size[0][0]
-            desired_size = max(150, 0.33 * best_frame.shape[1])
-            font_scale = desired_size / text_width
-            cv2.putText(
+            logger.info("Jetzad")
```

Comment:
Should this be removed?

Reply:
Yes, this was an artifact from debugging. Sorry